From b4fd66d346f15445c3bcfeab5d8932e059070f34 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 27 Aug 2018 11:04:35 +0800
Subject: [PATCH 4445/5242] MLK-19338 ARM64: dts: freescale: imx8qm: move mu
 interrupt controller to gic

commit  ea5d31fefcc30f0011af20fdcdeb748d907deac9 from
https://source.codeaurora.org/external/imx/linux-imx.git

To support lowest power mode for suspend, if no wakeup source
from non-secure partition is enabled, IRQSTEER can be powered
off when suspend, so non-secure linux OS needs to pass WU
irqchip wakeup source info to ATF, as MU is always enabled
as wakeup source, and it is a system level resource, so no
need to have it in WU domain.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
(cherry picked from commit d45b2fd47417cc1c4e03616007271e07834cf415)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 5279e65..23c91e8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -142,7 +142,7 @@
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1c0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 177 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-parent = <&wu>;
+		interrupt-parent = <&gic>;
 		fsl,scu_ap_mu_id = <0>;
 		status = "okay";
 	};
-- 
1.7.9.5

